app/views/runs/_timeline.slim

```diff
@@ -1,5 +1,5 @@
 - scale_to ||= run.duration(timing).to_ms
-- if run.duration(timing).present? && !run.completed?(timing)
+- if run.duration(timing).blank? || !run.completed?(timing)
```

Comment:
On second look this line seems like it was always redundant (or contradictory), as `Run#completed?` checks `duration(timing).present?` also.